tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_Translation.cpp

```diff
@@ -46,72 +54,114 @@ void test_translation() {
 
   const FoftPtr& f_of_t = f_of_t_list.at("translation");
 
-  const CoordinateMaps::TimeDependent::Translation<Dim> trans_map{
-      "translation"};
-  // test serialized/deserialized map
-  const auto trans_map_deserialized = serialize_and_deserialize(trans_map);
-
   UniformCustomDistribution<double> dist_double{-5.0, 5.0};
   std::array<double, Dim> point_xi{};
+  std::array<double, Dim> center{};
   fill_with_random_values(make_not_null(&point_xi), make_not_null(&gen),
                           make_not_null(&dist_double));
+  fill_with_random_values(make_not_null(&center), make_not_null(&gen),
+                          make_not_null(&dist_double));
+  const CoordinateMaps::TimeDependent::Translation<Dim> translation_map{
+      "translation"};
+  const CoordinateMaps::TimeDependent::Translation<Dim> radial_translation_map{
+      "translation",
+      std::make_unique<MathFunctions::Gaussian<1, Frame::Inertial>>(gaussian),
```

Comment:
You should be able to just pass the constructor arguments of Gaussian to `make_unique` and it'll construct the object for you rather than copying an existing object.